apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -165,7 +170,8 @@ class ManageStudentsTable extends Component {
         position: 0
       }
     },
-    showCopiedMsg: false
+    showCopiedMsg: false,
+    sectionCodeDialog: false
```

Comment:
NIT: `sectionCodeDialog` here sounds like an element/object. Consider changing it to a more truthy sounding statement. i.e. `showSectionCodeDialog`

Reply:
✔️